modules/rest-api/pages/rest-configure-memory.adoc

```diff
@@ -59,11 +84,12 @@ curl  -v -X POST http://10.144.220.101:8091/pools/default \
 -d 'cbasMemoryQuota=1024'
 ----
 
-
 == See Also
 
-For information on the maximum memory allocation permitted for a node, see xref:learn:buckets-memory-and-storage/memory.adoc#service-memory-quotas[Service Memory Quotas].
+* For information about the maximum memory allocation permitted for a node, see xref:learn:buckets-memory-and-storage/memory.adoc#service-memory-quotas[Service Memory Quotas].
+
+* For the other aspects of the provisioning of a single-node cluster, see xref:rest-api:rest-initialize-node.adoc[Initializing a Node], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
```

Comment:
`* For the other aspects of the provisioning of a single-node cluster using the REST API, see xref:rest-api:rest-initialize-node.adoc[Initializing a Node], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].`